canonical-web-and-design/snapcraft.io
# Publisher - Snap list

Fixes https://github.com/canonical-web-and-design/snapcraft.io/issues/2141

This will hopefully be temporary until we implement the new dashboard designs.

## QA
- Pull the branch
- `./run`
- Visit https://0.0.0.0:8004/snaps
- Ensure the list of snaps looks ok
- clicking on the name/ icon goes to the listing page
- To test various states:
  - No snaps - `/webapp/publisher/snaps/views.py:102` add 
```
user_snaps = {}
registered_snaps = []
```
  - 1 snap - edit `user_snaps = {}` to `user_snaps = {"snap_name": user_snaps["snap_name"]}`
  - new snap - below `user_snaps = {"snap_name": user_snaps["snap_name"]}` add `user_snaps["snap_name"]["is_new"] = True`
